gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Process/Traversal/DriverRemoteConnection/GraphTraversalTests.cs

```diff
@@ -201,22 +204,21 @@ public void ShouldUseBindingsInTraversal()
         [Fact]
         public void ShouldUseOptionsInTraversal()
         {
-            // smoke test to validate serialization of OptionsStrategy. no way to really validate this from an integration
-            // test perspective because there's no way to access the internals of the strategy via bytecode
             var connection = _connectionFactory.CreateRemoteConnection();
             var options = new Dictionary<string,object>
             {
                 {"x", "test"},
                 {"y", true}
             };
             var g = AnonymousTraversalSource.Traversal().WithRemote(connection);
-
-            var b = new Bindings();
+            
             var countWithStrategy = g.WithStrategies(new OptionsStrategy(options)).V().Count().Next();
             Assert.Equal(6, countWithStrategy);
+            
+            Assert.Equal(ResponseStatusCode.ServerTimeout, Assert.Throws<ResponseException>(() => 
```

Comment:
(nitpick) I think it's easier to read and understand this when the two asserts are separated:

```c#
var responseException = Assert.Throws<ResponseException>(() =>
    g.With("y").With("x", "test").With(Tokens.ArgsEvalTimeout, 10).Inject(1)
        .SideEffect(Lambda.Groovy("Thread.sleep(10000)")).Iterate());
Assert.Equal(ResponseStatusCode.ServerTimeout, responseException.StatusCode);
```